source/_posts/Creating-obfuscated-code-in-python-to-make-them-unreadable.md

```diff
@@ -0,0 +1,60 @@
+
+---
+title: Creating obfuscated code in python to make them unreadable 
+date: 2020-08-23 10:00:00
+author: Himanshu Garg
+categories:
+  - ["Scripting"]
+tags:
```

Comment:
put 3 and 4 keyword from given keywords list and use in content also

NAYAN
Traffic monitoring
AI
Livelihood
Driver analytics
Violation mapping
Better commute
Monitor infrastructure defects
Safer roads
Road Safety
Traffic Violation
Artificial Intelligence
India
Smart City
Safety
Data Protection
PARKING DETECTION,
VEHICLE OCCUPANCY DETECTION
NAYANINDIA
USER INTERACTION
NAYAN IP
NAYAN DATA PROTECTION
Road User
Public Road
Vehicle
Dangers
MACHINE LEARNING

Reply:
done, please check